SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1273,7 +1273,8 @@ void FSpatialGDKEditorToolbarModule::OnAutoStartLocalDeploymentChanged()
 		{
 			// Bind the TryStartSpatialDeployment delegate if autostart is enabled.
 			UEditorEngine::TryStartSpatialDeployment.BindLambda([this](FString ForceSnapshot) {
-				if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
+				if (GetDefault<USpatialGDKEditorSettings>()->bAutoStartLocalDeployment
+					&& GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
```

Comment:
I think that with this extra check in here we could now make this logic a bit easier and simply bind it once instead of binding and unbinding whenever the settings change. @improbable-valentyn  @mironec   what do you think?

though that could be done in a seaprate PR. the main thing we wanted to fix has been fixed 🎉

Reply:
I did find it weird that we bind and unbind everytime...